### DISCUSSION TITLE
Account banned because of locking

### DISCUSSION THREAD

**Discussion by FaisalAhmedAlghamdi** [SuppliedOrange/VALORANT-Instalocker#15] [General]
Title, its not safe to use instalockers anymore, i locked only like 5 times and i got banned a week later.

**Comment by SuppliedOrange** [comment]
Hello, you are officially the first person to report this in the past few years of this application's lifetime. Neither I nor my friends have been banned for using it this week.

Please be specific about your ban and share a screenshot of possible. Did you get any warning bans for API abuse? Were you using this instalocker?

**Comment by FaisalAhmedAlghamdi** [comment]
VAN 152, Im working with riot to see what caused the ban but im very sure its the instalocker, I will report again when riot responds in 4 days.

**Comment by SuppliedOrange** [comment]
@FaisalAhmedAlghamdi please do, because people don't usually get banned immediately like this. If this was because of the instalocker, you will get a 7 day warning ban for API abuse. VAN 152 might be because of actual cheats on your system or stuff like the interception driver actively doing stuff etc. Let me know.
